modules/blocks/chain.js

```diff
@@ -495,6 +492,20 @@ Chain.prototype.applyBlock = function (block, broadcast, cb, saveBlock) {
 	});
 };
 
+/**
+ * Broadcast reduced block to increase network performance.
+ * @param {Object} reducedBlock reduced block
+ * @param {boolean} broadcast Indicator that block needs to be broadcasted
+ * @throws {string} Error description
+ */
+Chain.prototype.broadcastReducedBlock = function (reducedBlock, broadcast) {
+	try {
```

Comment:
good catch, at the beginning bus.message was not in the test scope and try/catch warned about that. There is no reason to keep the warning because bus will be loaded from start.